DopplerFiles/Properties/launchSettings.json

```diff
@@ -12,7 +12,7 @@
         "IIS Express": {
             "commandName": "IISExpress",
             "launchBrowser": true,
-            "launchUrl": "test/anonymous",
+            "launchUrl": "manager/12345",
```

Comment:
Maybe `manager` is not the best prefix.

If I am not wrong, this API already has a prefix, something like `https://apis.fromdoppler.com/files/` (in production), so, what do you think about using a [schema similar to billing API's one](https://github.com/FromDoppler/doppler-billing-api/blob/baabb89ef43d53a36e5e76bae617fb687c0b7999/Billing.API/Controllers/InvoiceController.cs#L38)?

```
https://apis.fromdoppler.com/files/accounts/doppler/{clientId}/
```

In the future, It will also allow dealing with ids from other products, not only Doppler.

Reply:
Do you think "manager" could conflict with other services?